server/src/main/scala/actors/CrunchStateActor.scala

```diff
@@ -120,11 +121,9 @@ class CrunchStateActor(initialMaybeSnapshotInterval: Option[Int],
     case DeleteSnapshotsSuccess(_) =>
       logInfo(s"Purged snapshots")
 
-    case "complete" =>
-      logInfo("Received complete")
+    case StreamCompleted => log.warn("Received shutdown")
```

Comment:
another candidate for StreamFailure?

Reply:
in this case it won't get a failure message because we're not using it as an ACKing actor sink